src/plugins/credential_management/public/components/create_credential_wizard/create_credential_wizard.tsx

```diff
@@ -8,3 +8,207 @@
+          <EuiDescribedFormGroup
+            title={<h3>Credential Type</h3>}
+            description={
+              <div>
+                <p>
+                  The type of credential that you want to create{' '}
+                  <EuiLink href="#/display/text">
+                    <strong>Credential Types Supported</strong>
+                  </EuiLink>
+                </p>
+                <ul>
+                  <li>
+                    For <b>username_password_credential</b> type: this type can be used for{' '}
+                    credentials in format of username, password.{' '}
+                  </li>
+                  <li> Ex: Opensearch basic auth </li>
+                </ul>
+                <ul>
+                  <li>
+                    For <b>aws_iam_credential</b> type: this type can only be used for{' '}
+                    aws iam credential, with aws_access_key_id,{' '}
+                    aws_secret_access_key, and region (optional) 
+                  </li>
+                </ul>
+              </div>
+            }
+          >
+            <EuiFormRow label="Credential Type">
+              <EuiSelect
+                onChange={(e) => this.setState({ credentialType: e.target.value })}
+                options={[
+                  { value: USERNAME_PASSWORD_TYPE, text: 'Username and Password Credential' },
+                  { value: 'aws_iam_credential', text: 'AWS IAM Credential' },
+                ]}
+              />
+            </EuiFormRow>
+            <EuiFormRow label="User Name">
+              <EuiFieldText
+                placeholder="Your User Name"
+                value={this.state.user_name || ''}
+                onChange={(e) => this.setState({ user_name: e.target.value })}
+              />
+            </EuiFormRow>
+            <EuiFormRow label="Password">
+              <EuiFieldText
+                placeholder="Your Password"
+                value={this.redact(this.state.password.length) || ''}
+                onChange={(e) => this.setState({ password: e.target.value })}
+              />
+            </EuiFormRow>
+            <EuiFormRow label="Upload Credential File">
+              <EuiFilePicker />
+            </EuiFormRow>
+          </EuiDescribedFormGroup>
+          <EuiButton type="submit" fill onClick={this.createCredential}>
+            Create
+          </EuiButton>
+        </EuiForm>
+      </EuiPageContent>
+    );
+  }
+
+  removeToast = (id: string) => {
+    this.setState((prevState) => ({
+      toasts: prevState.toasts.filter((toast) => toast.id !== id),
+    }));
+  };
+  render() {
+    const content = this.renderContent();
+
+    return (
+      <>
+        {content}
+        <EuiGlobalToastList
+          toasts={this.state.toasts}
+          dismissToast={({ id }) => {
+            this.removeToast(id);
+          }}
+          toastLifeTimeMs={6000}
+        />
+      </>
+    );
+  }
+
+  redact = (len: number) => {
+    return '*'.repeat(len);
+  };
```

Comment:
see https://elastic.github.io/eui/#/forms/form-controls#password-field

Reply:
Resolved in #10